recipe/symfony-flex.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Deployer;
+
+require_once __DIR__ . '/symfony3.php';
+
+/**
+ * Symfony Flex Configuration
+ */
+
+//No need to clear anything
+set('clear_paths', []);
+
+// Symfony shared dirs
+set('shared_dirs', ['var/log', 'var/sessions']);
+
+// Symfony writable dirs
+set('writable_dirs', ['var/cache', 'var/log', 'var/sessions']);
+
+//File for DotEnv if not using env vars
+set('shared_files', ['.env']);
```

Comment:
And btw the dotenv component is in require-dev when we do a fresh install ... I'm not sure if it's a good idea

Reply:
Actually there is a conversation on moving dotenv to require https://github.com/symfony/symfony/issues/25643